lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -722,7 +788,10 @@ def test_cache_map_traverse_cmap(
             )
             tb.cache_map(tar_dir)
 
-            if str(resolve_path).endswith("subdir15"):
+            # Hack for absolute paths
+            if str(resolve_path).endswith("subdir15") or str(location).endswith(
+                "f1412_sym"
+            ):
                 resolve_path = tar_dir / resolve_path
```

Comment:
Why do we need this hack?  Why can't we just set the values of `resolve_path` appropriately in the parametrize list so that they match properly without modification?

Reply:
`f1412` is resolving to an absolute path.. which is build by `tmp_path` of tests.

Reply:
I now understand why this code is here.  It would be helpful if the code comment conveyed more of the "why".  Also, I have a [suggestion below](https://github.com/distributed-system-analysis/pbench/pull/3396#discussion_r1204872265) for making the code more general and focused, but Dave offered what might be a better [suggestion above](https://github.com/distributed-system-analysis/pbench/pull/3396#discussion_r1198183679).